pkgs/desktops/gnome-3/core/tracker-miners/default.nix

```diff
@@ -51,11 +86,33 @@ in stdenv.mkDerivation rec {
     };
   };
 
+  postPatch = ''
+    # audio/flac is the proper MIME type for FLAC audio (fixes GNOME Music):
+    substituteInPlace src/tracker-extract/10-flac.rule --replace audio/x-flac audio/flac
```

Comment:
It might make more sense to investigate why the GStreamer extractor does not work. Maybe we need to add `gst-plugins-good` or something.

Reply:
This still is an upstream bug.

Reply:
https://github.com/NixOS/nixpkgs/issues/39545

Reply:
But intentional one, see https://git.gnome.org/browse/tracker-miners/commit/?id=b830fdf96a97e6f37a25a4bb58106748c585d8e3.

Reply:
This doesn't make any sense :-) libflac extractor does work, and this commit deliberately breaks it. Would make more sense to put it behind a switch.

Reply:
Well, GStreamer extractor is supposed to work better according to the commit. If we manage to fix the bug, that is.

Reply:
Maybe open an upstream bug about this as well.